tgui/packages/tgui/interfaces/GasAnalyzer.tsx

```diff
@@ -1,22 +1,47 @@
 import { useBackend } from '../backend';
 import { GasmixParser } from './common/GasmixParser';
 import type { Gasmix } from './common/GasmixParser';
+import { Button, Flex, LabeledList, Section, Stack, Box } from '../components';
 import { AtmosHandbookContent, atmosHandbookHooks } from './common/AtmosHandbook';
 import { Window } from '../layouts';
-import { Section } from '../components';
+
+type GasmixHistory = {
+  allGasmixes: Gasmix[];
+};
 
 export type GasAnalyzerData = {
   gasmixes: Gasmix[];
+  autoUpdating: boolean;
+};
+
+export type GasAnalyzerHistoryData = {
```

Comment:
you shouldn't need to export this

Reply:
What about `GasAnalyzerData`? If it's not used in another file, it shouldn't have export, right?